src/lib/ELF/Drude.m

```diff
@@ -18,13 +22,18 @@
         w_at_q = sqrt(w_at_q_square);
     end
     
+    
+    
     %divisor = (w_global*w_global - w_at_q*w_at_q)*(w_global*w_global - w_at_q*w_at_q) + w_global*w_global*gamma * gamma;
     mm = bsxfun(@minus,w_global.^2,w_at_q.^2);
     divisor = mm.^2 + repmat(w_global.^2*gamma^2,1,sq);
 % 	re_oneovereps =1.0+(omega0*omega0)*(w_global*w_global - w_at_q*w_at_q) / divisor;   
     eps_re = mm ./ divisor;
     eps_im = repmat(w_global*gamma,1,sq) ./ divisor;
     
-end
-
-
+    if isIonization
+        eps_re(w_global<w_at_q,:)=0;
```

Comment:
Look through the Vos article, peak position for Drude function is not there. It’s somwhat like `sqrt(A+wp^2)`